searchcore/src/vespa/searchcore/proton/server/documentretriever.cpp

```diff
@@ -82,14 +82,15 @@ ::DocumentRetriever(const DocTypeName &docTypeName,
       _attr_manager(attr_manager),
       _doc_store(doc_store),
       _possiblePositionFields(),
-      _attributeFields(),
+      _attributeFields(document::Field::Set::Builder().build()),
```

Comment:
consider adding a static convenience to make this case simpler.

Reply:
emptySet